client/pebble.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2023 Canonical Ltd
```

Comment:
2023, nice! :-)

IMO we should name this `shutdown.go`. We've tried to avoid using the name "Pebble" itself in contexts like this, as everything is Pebble here, after all. So IMO `shutdown.go`, but we could think of a broader name (or just call it shutdown.go now, and wait until there are multiple things that apply to think of a better name.